functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,21 @@
  */
 struct BeerOrganizer
 {
+    BeerOrganizer()
+        : state{BeerBrand::None}
+    {}
+    BeerBrand operator()()
+    {
+        int nextIndex = getNextIndex();
+        state = (0 == nextIndex) ? BeerBrand::HoeGaarden : static_cast<BeerBrand>(nextIndex);
```

Comment:
please specify range with static constexpr variablies: "to" and "from"

Reply:
confused but done